tests/acceptance/administrator/g07/Configuration/ConfigurationShowDiscontinuedProductsCest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+/**
+ * @package     redSHOP
+ * @subpackage  Cest
+ * @copyright   Copyright (C) 2008 - 2020 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+use AcceptanceTester\CategoryManagerJoomla3Steps;
+use AcceptanceTester\ProductManagerJoomla3Steps;
+use Configuration\ConfigurationSteps;
+use Configuration\ProductsConfigurationSteps;
+
+/**
+ * Class ConfigurationShowDiscontinuedProductsCest.
+ * @since 3.0.2
+ */
+class ConfigurationShowDiscontinuedProductsCest
+{
+	/**
+	 * @var \Faker\Generator
+	 * @since 3.0.2
+	 */
+	protected $faker;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $productName;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $randomCategoryName;
+
+	/**
+	 * @var int
+	 * @since 3.0.2
+	 */
+	protected $productNumber;
+
+	/**
+	 * @var int
+	 * @since 3.0.2
+	 */
+	protected $price;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $optionYes;
+
+	/**
+	 * @var string
+	 * @since 3.0.2
+	 */
+	protected $optionNo;
+
+	/**
+	 * ConfigurationShowDiscontinuedProductsCest constructor.
+	 * @since 3.0.2
+	 */
+	public function __construct()
+	{
+		$this->faker              = Faker\Factory::create();
+		$this->productName        = $this->faker->bothify('Testing productName_?##?');
+		$this->randomCategoryName = $this->faker->bothify('Category Demo ?##?');
+		$this->productNumber      = $this->faker->numberBetween(999, 9999);
+		$this->price              = $this->faker->numberBetween(99, 199);
+		$this->optionYes          = "Yes";
+		$this->optionNo           = "No";
+	}
+
+	/**
+	 * @param AcceptanceTester $I
+	 * @throws Exception
+	 * @since 3.0.2
+	 */
+	public function _before(AcceptanceTester $I)
+	{
+		$I->doAdministratorLogin();
+	}
+
+	/**
+	 * @param ConfigurationSteps $I
+	 * @param $scenario
+	 * @throws Exception
+	 * @since 3.0.2
+	 */
+	public function checkProductDiscontinueYes(ConfigurationSteps $I, $scenario)
+	{
+		$I->wantTo ('I Want To Configuration Product Accessory Yes');
```

Comment:
I think here it would be  "I Want To Configuration Show Discontinued Products Yes".